rest-server/test/etcd2Store.js

```diff
@@ -22,6 +22,13 @@ const db = dbUtility.getStorageObject('etcd2', {
   'hosts': etcdHosts,
 });
 describe('etcd2 get function test', () => {
+  afterEach(function() {
+    if (!nock.isDone()) {
+      //TODO: Revamp this file and enable the following error.
+      //this.test.error(new Error('Not all nock interceptors were used!'));
+      nock.cleanAll();
+    }
+  });
 
```

Comment:
Make this a global function and call in each test to avoid code dup?

Reply:
Thanks! Will do.